lib/contracts/testWrapper.ts

```diff
@@ -0,0 +1,45 @@
+"use strict";
```

Comment:
Does the test wrapper suppose to be under test dir ?
Is it part of the lib ? why ?

Reply:
Because it is typescript, it is not currently possible to pull this into the test folder -- mocha doesn't understand it.  It may be possible to convert all of the tests to typescript, but that needs to be a separate PR as it would take time and add complexity to this PR.

It can't be excluded from the NPM package because we have chosen to include tests in the package.

I have created an issue for this:  https://github.com/daostack/arc.js/issues/99

Reply:
how about moving that to contracts/test/ or lib/test/ dir ?

Reply:
Done